tt_metal/hw/inc/dataflow_api.h

```diff
@@ -226,6 +226,18 @@ FORCE_INLINE T get_common_arg_val(int arg_idx) {
  */
 #define get_compile_time_arg_val(arg_idx) KERNEL_COMPILE_TIME_ARG_##arg_idx
```

Comment:
Like the compile time arg header I think the implementation of this should be replaced with the new way (index into the array)

Reply:
this is still unresolved